plugins/woocommerce-admin/client/dashboard/components/settings/general/store-address.tsx

```diff
@@ -28,10 +28,10 @@ type Option = { key: string; label: string };
 /**
  * Form validation.
  *
- * @param {Object} locale The store locale.
  * @return {Function} Validator function.
  */
-export function getStoreAddressValidator( locale: Locale = {} ) {
+// Note: param was removed because its not used, callers still assume its needed. TODO: Fix this.
+export function getStoreAddressValidator(/* locale: Locale = {} */) {
```

Comment:
Note that in places where I felt I should leave the info behind for the person who should eventually fix a problem I've used comments instead of removing the linted code.